dom5.ts

```diff
@@ -38,6 +38,15 @@ export function hasAttribute(element: Node, name: string): boolean {
   return getAttributeIndex(element, name) !== -1;
 }
 
+export function hasSpaceSeparatedAttributeValue(element: Node, name: string, value: string): boolean {
```

Comment:
Are there other delimiters we might want to use with attributes? I'm all for adding that abstraction when we first need it, but until then maybe YAGNI?